pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -664,7 +667,7 @@ func (k *Kubernetes) Deploy(komposeObject kobject.KomposeObject, opt kobject.Con
 			if err != nil {
 				return err
 			}
-			log.Infof("Successfully created PersistentVolumeClaim: %s", t.Name)
+			log.Infof("Successfully created PersistentVolumeClaim: %s of size %s. PVC's will not work unless a PersistentVolume has been created.", t.Name, PVCRequestSize)
```

Comment:
I thought it was there before, but missed @cdrage's comment.
We can make it a bit shorter, but we have to mention that you don't have to do anything if you have dynamic provisioning, otherwise it will be confusing.

Another version:
`PersistentVolumeClaim of size 100Mi created. If your cluster has dynamic storage provisioning, you don't have to do anything. Otherwise you have to create PersistentVolume to make PVC work.`

This a bit shorter than original and says everything.
thoughts @cdrage ?